### ISSUE TITLE
Icon A11y Pass

### ISSUE THREAD

**Issue by TestAccessibilityGIT** [OfficeDev/office-ui-fabric-react#7852]
Tracking all of the issues brought up with the 1/30 accessibility pass on Icon control 
Details on individual bugs can be found on [SharePoint](https://microsoft-my.sharepoint.com/:x:/r/personal/v-amaver_microsoft_com/_layouts/15/Doc.aspx?sourcedoc=%7B164d2d05-98d3-4f57-82f0-17c7345061ff%7D&action=default&uid=%7B164D2D05-98D3-4F57-82F0-17C7345061FF%7D&ListItemId=1724&ListId=%7B395E1ECB-5002-440E-81C5-6A090E98077C%7D&odsp=1&env=prodbubble) 
  
 - [ ] Bug1: [Accessibility][ Describing content, controls, or links]: Images must have alternate text." 
 - [ ] Bug2: [Accessibility][Usable]: User is not able to move to next icons using Up/Down arrow keys in  narrator's scan mode. 
 
 
 Component: Icon 
@micahgodbolt

**Comment by dzearing** [comment]
I'm not sure bug 1 true. Icons I believe should be role=presentation in most cases. The outer container rendering them should have the properly descriptive aria tags.

Bug 2 is not a real issue either. Icon is a graphical glyph to be used within an interactive element that has the right ariaLabels.

**Comment by dzearing** [comment]
@TestAccessibilityGIT if you disagree, please let us know. @cschlechty I think would agree here, but correct me if not.
